packages/wrangler/src/__tests__/navigator-user-agent.test.ts

```diff
@@ -110,6 +110,7 @@ describe("defineNavigatorUserAgent is respected", () => {
 				exports: [],
 			},
 			path.resolve("dist"),
+			// @ts-expect-error Ignore the requirement for passing undefined values
```

Comment:
Nit: Just wanted to avoid `@ts-expect-error`. This might also make the test clearer which options the tests really cares.

```ts
{
  // Extracted from the original options
  ...defaultBundleOptions,
  defineNavigatorUserAgent: false,
}
```